### PR TITLE
ENG-619 upload client jar to maven central

### PR DESCRIPTION
**Pull request by vtkstef**

### CODE REVIEW

File: upload.gradle [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+               scm {
+                   url 'git@github.com:VoltDB/voltdb.git'
+                   connection 'scm:git@github.com:VoltDB/voltdb.git'
+                   developerConnection 'scm:git@github.com:VoltDB/voltdb.git'
+               }
+
+               licenses {
+                   license {
+                       name 'GNU Affero General Public License Version 3'
```
Comment (akhanzode) [inline upload.gradle]: Is this correct lic?

File: upload.gradle [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    include 'org/voltdb/utils/SerializationHelper.java'
+    include 'org/voltdb/utils/RowWithMetaData.java'
+    include 'org/voltdb/utils/CSVDataLoader.java'
+    include 'org/voltdb/utils/CSVBulkDataLoader.java'
+    include 'org/voltdb/utils/CSVTupleDataLoader.java'
+    include 'org/voltdb/VoltOverflowException.java'
+    include 'org/voltdb/VoltTable.java'
+    include 'org/voltdb/VoltTableRow.java'
+    include 'org/voltdb/VoltType.java'
+    include 'org/voltdb/VoltTypeException.java'
```
Comment (akhanzode) [inline upload.gradle]: Thought we were not packaging sources.

File: upload.gradle (hunk 1 above)
Comment (vtkstef) [inline upload.gradle]: Sonatype won't let us publish to maven central without sources. I talked it over with John H and we decided to publish the sources under the Affero license.

**Comment by akhanzode** [comment]
this looks good. as long as folks know how to publish new versions (wiki) I am good with merging.

**Comment by rmorgenstein** [comment]
Instead of writing a wiki, I would prefer you to write a jenkins job.  Make it start with 'kit-job' and add a call to it from kit-tools-publish-kit-masterjob.

**Comment by akhanzode** [comment]
Should we package Export client classes as well so people can write custom connectors without having server.??

**Pull request merged**